### DISCUSSION TITLE
有没有大佬在使用lucky做本地反代服务？如何和natter配合使用？

### DISCUSSION THREAD

**Discussion by Asakushen** [MikeWang000000/Natter#118] [General]
我一直用的lucky做本地反代，比如本地反代端口是1234，如何实现打洞后的端口转发到lucky机器上的1234端口上？
顺便 我比较抽象开了三台虚拟机，一台打洞，一台openwrt做旁路由，一台Ubuntu跑pt和docker，目前给qb打洞，想顺便也给lucky打一个

**Comment by MikeWang000000** [comment]
Lucky 目前也支持打洞了，或许可以直接参照他们的文档操作。

据 Lucky 作者所说，Lucky 的打洞功能也是参考 Natter 实现的，所以我觉得本质上应该没有什么区别。
